terraform/locals.tf

```diff
@@ -105,8 +105,6 @@ locals {
     local.lb_ip,
   ) : ""
 
-  async_prov_mode = !local.assign_weblogic_public_ip && !var.is_bastion_instance_required ? "Asynchronous provisioning is enabled. Connect to each compute instance and confirm that the file /u01/data/domains/${format("%s_domain", local.service_name_prefix)}/provCompletedMarker exists. Details are found in the file /u01/logs/provisioning.log." : ""
```

Comment:
Did you check if this local variable is used elsewhere?

Reply:
I checked it. It is used only in outputs.tf. Thats why I have removed it as the condition didn't make sense.